storage/test/authz_models.go

```diff
@@ -49,50 +49,57 @@ func TestWriteAndReadAuthorizationModel(t *testing.T, datastore storage.OpenFGAD
 		),
 	}
 
-	if diff := cmp.Diff(expectedModel, model.TypeDefinitions, cmpOpts...); diff != "" {
+	if diff := cmp.Diff(got.TypeDefinitions, model.TypeDefinitions, cmpOpts...); diff != "" {
```

Comment:
Let's just compare the returned model against the provided model in their entirety, instead of checking only the type defs.

Every field of `got` and `model` should be equal, and so we may as well compare them directly. It will increase our test confidence by doing this as well, because we'll be making sure the fields match in their entirety.

Reply:
Good call. Updated. Thanks!